src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -159,6 +159,10 @@
             durationCell.textContent = 'n/a';
             row.appendChild(durationCell);
 
+            request.liveDurationHandle = setInterval(function() {
+				durationCell.textContent = (new Date() - request.start) + 'ms';
+			}, 100);
```

Comment:
Looks like there is an indentation issue here (tabs?).

Reply:
indeed